tests/TestRvsdgs.hpp

```diff
@@ -855,6 +855,141 @@ class GammaTest final : public RvsdgTest {
 	jlm::rvsdg::gamma_node * gamma;
 };
 
+/** \brief GammaTest2 class
+ *
+ * This function sets up an RVSDG representing the following function:
+ *
+ * \code{.c}
```

Comment:
Not fluent enough to read construction of RVSDG to tell if it is a true representation of the this C code